### PR TITLE
Remove pkg and git analyzers

### PR DESCRIPTION
**Pull request by GiasemiSh**
Remove pkg and git analyzers from the yaml configuration files and
add the functionality in the build scripts.
Also add the option to skip the generation of the html pages in qmstr-reporter-html
in comment.

### CODE REVIEW

File: demos/calc/qmstr.yaml [hunk 1]
```diff
@@ -48,5 +39,6 @@ project:
     - reporter: qmstr-reporter-html
       name: "HTML Reporter"
       config:
+        #generatehtml: "no"
```
[the remark this answers is not in the document]
Comment (GiasemiSh) [reply demos/calc/qmstr.yaml]: @senaingrid Hadn't seen it sorry. Just approved it, you can merge it now

File: demos/jsonc/build.sh [hunk 2]
```diff
@@ -19,7 +19,7 @@ echo "Waiting for qmstr-master server"
 eval $(qmstrctl start --wait)
 echo "master server up and running"
 
-qmstrctl create project:jsonc
+qmstrctl create project:jsonc --version $(cd jsonc && git describe --always)
```
[the remark this answers is not in the document]
Comment (GiasemiSh) [reply demos/jsonc/build.sh]: Good catch

File: demos/calc/build.sh [hunk 3]
```diff
@@ -17,14 +17,16 @@ echo "Waiting for qmstr-master server"
...
 echo "master server up and running"
 
-qmstrctl create package:calc
+qmstrctl create package:calc --version 00000
 
 pushd Calculator
 make clean
 popd
 
 qmstr --container qmstr/calcdemo -- make -j4
```
[the remark this answers is not in the document]
Comment (hemarkus) [reply demos/calc/build.sh]: Delay commands in scripts/programs are never a good idea. @senaingrid can you check if a simple `sync` will fix this.

**Comment by GiasemiSh** [comment]
Please review again @senaingrid @hemarkus

File: demos/calc/build.sh [hunk 4]
```diff
@@ -18,13 +18,14 @@ eval $(qmstrctl start --wait)
...
 
 qmstrctl create package:calc
-
 pushd Calculator
 make clean
 popd
 
 qmstr --container qmstr/calcdemo -- make -j4
 
+qmstrctl connect package:calc file:Calculator/libcalc.a file:Calculator/calcs file:Calculator/libcalc.so file:Calculator/calc
```
Comment (hemarkus) [inline demos/calc/build.sh]: you need to add `qmstrctl wait` before trying to connect

Comment (GiasemiSh) [reply demos/calc/build.sh]: @hemarkus  Done

**Pull request merged**